jujugui/static/gui/src/app/components/profile/navigation/_navigation.scss

```diff
@@ -1,30 +1,29 @@
 .profile-navigation {
   color: $color-dark;
+  padding: 0 1rem;
 
-  &__list-item {
-    list-style: none;
+  .p-list__item {
     cursor: pointer;
 
-    &:nth-child(4) {
-      border-top: 1px solid #cdcdcd;
-      padding-top: 12px;
-      margin-right: 24px;
+    .profile__content &.cloud-credentials {
```

Comment:
The extra class is needed for adequate specificity otherwise this rule is overwritten - this is because adding the `v1` namespace gives the default Vanilla styling added clout. 

The alternative is an `!important` - while not idea, I consider this approach the lesser of two evils.